public/pages/AnomalyCharts/containers/AnomalyDetailsChart.tsx

```diff
@@ -344,6 +348,59 @@ export const AnomalyDetailsChart = React.memo(
       zoomRange.endDate,
     ]);
 
+
+    const customAnomalyContributionTooltip = (details?: string) => {
+      const anomaly = details ? JSON.parse(details) : undefined;
```

Comment:
this would include anomaly results with grade 0 too, right? Suggest to rename to `anomalyResults` for clarity

Reply:
no it won't. it's already been filtered in `generateContributionAnomalyAnnotations`